### ISSUE TITLE
Investigate needed funcitonality for character modding

### ISSUE THREAD

**Issue by enenra** [enenra/space-engineers-utilities#22]
There's some specific stuff for it and I should look into it properly, then see what can be done.

https://forum.keenswh.com/threads/guide-importing-the-astronaut-into-blender.7382628/

Some findings about export by Stollie: https://discordapp.com/channels/125011928711036928/161758345856811008/674725223542685701

https://steamcommunity.com/sharedfiles/filedetails/?id=433069217&searchtext=character

https://forum.keenswh.com/threads/guide-custom-gesture-animations-video-tutorial.7399208/

**Comment by enenra** [comment]
```
SE_main_root {very important, its base parent bone for whole rig}
FrontJetpackDummy
BackJetpackDummy
ThirdCameraDummy {parented to SE_main_root }
HeadDummy {parented toSE_RigHead}
RightLightDummy {parented toSE_RigHead}
LeftLightDummy {parented toSE_RigHead}
```

**Comment by enenra** [comment]
This was implemented into 0.9.0 by @cstahlhut .
